src/pages/workspaces/[workspaceSlug]/pipelines/[pipelineCode]/index.tsx

```diff
@@ -171,6 +190,40 @@ const WorkspacePipelinePage: NextPageWithLayout = (props: Props) => {
                 }
               </RenderProperty>
             </DataCard.FormSection>
+            <DataCard.FormSection
+              title={t("Scheduling")}
+              onSave={pipeline.permissions.update ? onSave : undefined}
+              collapsible={false}
+            >
+              <RenderProperty<any> id="enableScheduling" label={t("Enabled")}>
+                {(property, section) => (
+                  <Switch
+                    name="enable_schedule"
+                    checked={isSchedulingEnabled}
+                    onChange={() => setIsSchedulingEnabled((value) => !value)}
+                    disabled={!section.isEdited}
+                  />
+                )}
+              </RenderProperty>
+              <CronProperty
+                id="schedule"
+                accessor="schedule"
+                label={t("Cron expression")}
+                defaultValue={t("Manual")}
+                visible={isSchedulingEnabled}
+                required
+              />
+              <WorkspaceMemberProperty
+                id="recipients"
+                label={t("Repicients")}
```

Comment:
```suggestion
                label={t("Notification Repicients")}
```
I'm not sure for the label but I think we have to add the purpose of the list. Mail recipients, Notification recipients, ...